routes/error-tracker.js

```diff
@@ -49,6 +47,39 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  const location = '([^ \\n]+):(\\d+):(\\d+)';
+  const chromeStackTraceRegex = new RegExp(`^\\s*at (.+ )?(?:${location}|\\(${location}\\))$`, 'gm');
+  const mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
```

Comment:
You can hoist these regexes out of the function to avoid recreating them on every invocation.